sdk/Pulumi/Pulumi.csproj

```diff
@@ -29,6 +29,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.24.0" />
```

Comment:
Just for my own understanding (because I don't know much about dotnet), was this dependency just missing before, or is it not expected for dotnet projects to specify all its dependencies, but only the pinned ones?

Reply:
No need to specify transitive dependencies unless you need to change the version range.